extensions/modules/file/pom.xml

```diff
@@ -62,6 +62,12 @@
             <artifactId>j8fu</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.evolvedbinary.multilock</groupId>
+            <artifactId>multilock</artifactId>
+            <version>1.0.1</version>
```

Comment:
not in parent pom?

Reply:
I can put it the dependency in a different POM if that is better.
As I am not so familiar yet with Maven please be a little more decisive. Could you also point me to the pom file you think is best?

Reply:
I think @dizzzz means that as this is used in another module in eXist-db, with your change there would be two modules with the same dependency. Therefore the common dependency should be declared in the appropriate part of the parent pom and only reference in the module poms.

Reply:
So, the question remains: which pom.xml is the desired one?

Reply:
>  parent pom